modules/distribution-zones/src/integrationTest/java/org/apache/ignite/internal/rebalance/ItRebalanceDistributedTest.java

```diff
@@ -898,6 +899,15 @@ void testRebalanceRetryDelayConfiguration() throws Exception {
         assertThat(updateRebalanceRetryDelay(configuration, -1), willThrowWithCauseOrSuppressed(ConfigurationValidationException.class));
     }
 
+    @Test
+    void testCompatibilityPropertyNameRebalanceRetryDelayMsNameWasNotChanged() {
```

Comment:
This should be put in a unit test, not inside an integration test

Reply:
Moved to `DistributionZoneManagerConfigurationChangesTest`